build-aux/snap/snapcraft.yaml

```diff
@@ -176,7 +176,11 @@ parts:
       # copy in a pregenerated list of network address families so that the
       # parser gets built to support as many as possible even if glibc in
       # the current build environment does not support them
-      cp $SNAPCRAFT_PROJECT_DIR/build-aux/snap/local/apparmor/af_names.h .
+      if [ -d "$SNAPCRAFT_PROJECT_DIR/build-aux" ]; then
```

Comment:
Thanks for this! Maybe a small comment why this conditional exists? Somehting like: ` For some reason, some snapcraft version remove the "build-aux" folder and move the contents up when the data is uploaded.` ?

Reply:
I can remove it, but I don't know if that can break other ways of building this... That's why I preferred to be conservative.

Reply:
Oh! You mean "to add a comment"! Of course! One second.

Reply:
Done.